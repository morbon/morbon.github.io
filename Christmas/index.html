<!DOCTYPE html>
<html>
<meta charset="utf-8">
<head>
	<title>С Рождеством Христовым!</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Pattaya" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Caveat" rel="stylesheet">
</head>
<body>
	<div class="png-header">
		<p><img src="img/christmas.png"></p>
	</div>
	<div class="container">
		<h2>С Рождеством Христовым!</h2>
		<div id="cvg">
			<img src="img/merry-christmas-decor-with-penguin-11546226539e38wck73uy.png">
		</div>
		<p>На Рождество мы верим  в чудеса,<br>
			С надеждою глядим на небеса!<br>
			Молим, чтоб счастье шло к нам в дом!<br>
			Богатство сыпалось дождем!<br>
			Чтоб на душе была весна,<br>
			Сбывалась каждая мечта!<br>
		</p>
		<audio src="audio/Novogodnie+We+wish+you+a+merry+Christmas+We+wish+you+a+merry.mp3"></audio>
		<audio controls autoplay>
			<source src="audio/music.ogg" type="audio/ogg; codecs=vorbis">
				<source src="audio/music.mp3" type="audio/mpeg">
					<source src="audio/Novogodnie+We+wish+you+a+merry+Christmas+We+wish+you+a+merry.mp3">
					</audio>
				</div>

			</body>
			</html>