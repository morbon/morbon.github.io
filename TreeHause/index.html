<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<title>Treehouse home</title>
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/navigation.css">
	<link rel="stylesheet" href="css/fonts/fonts.css">
	<link rel="stylesheet" href="css/buttons/buttons.css">
	<link rel="stylesheet" href="css/titles/titles.css">
	<link rel="stylesheet" href="css/mobiles/mobi.css">
</head>
<body>
	<!----------------------- Navigation --------------------------->
	<div class="container">
		<div class="row">
			<div class="logo-holder">
				<a href="#">
					<img src="img/logo/logo-text.png" alt="Treehouse home"">
				</a>
			</div>
			<div class="navigation-holder" id="navigation">
				<ul class="navigation-list">
					<li href="#"><a href="#">Home</a></li>
					<li href="#"><a href="#">About</a></li>
					<li href="#"><a href="#">Blog</a></li>
					<li href="#"><a href="#">Portfolio</a></li>
					<li href="#"><a href="#">Contact</a></li>
				</ul>
			</div>
 
      <div class="navigation-button" id="nav-toggle">
      	SHOW NAVIGATION
      </div>

		</div>
	</div>
	<!----------------------- // Navigation --------------------------->

	<!--------------------------- Header ------------------------------>
	<div class="container-fluid">
		<div class="header">
			<div class="header-content">
				<h1 class="heading-1">Creative Digital Solutions</h1>
				<p class="subtitle margin-bottom-50 subtitle--header">Proin iaculis purus consequat sem cure.</p>
				<a class="button" href="#">View Portfolio</a>
			</div>
		</div>
	</div>
	<!------------------------ // Header ------------------------------>

	<!------------------- Professional Services ----------------------->
	<div class="container container-padding">
		<div class="heading-blok">
			<h2 class="heading-2">Professional Services</h2>
			<p class="subtitle">Proin iaculis purus consequat sem cure</p>
		</div>
		<div class="row">
			<div class="col-1-of-3">
				<!-- Service box -->
				<div class="service-box">
					<div class="service-box__icon">
						<img src="img/icon/icon-heart.png" alt="Heart">
					</div>
					<div class="service-box__header">beautiful designs</div>
					<p class="service-box__text">Proin iaculis purus consequat sem cure digni ssim. Donec 
					porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<!-- // Service box -->
			</div>
			<div class="col-1-of-3">
				<!-- Service box -->
				<div class="service-box">
					<div class="service-box__icon">
						<img src="img/icon/icon-mobile.png" alt="Heart">
					</div>
					<div class="service-box__header">responsive web</div>
					<p class="service-box__text">Proin iaculis purus consequat sem cure digni ssim. Donec 
					porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<!-- // Service box -->
			</div>
			<div class="col-1-of-3">
				<!-- Service box -->
				<div class="service-box">
					<div class="service-box__icon">
						<img src="img/icon/icon-graph.png" alt="Heart">
					</div>
					<div class="service-box__header">Marketing tools</div>
					<p class="service-box__text">Proin iaculis purus consequat sem cure digni ssim. Donec 
					porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<!-- // Service box -->
			</div>
		</div>
	</div>
	<!------------------- // Professional Services ----------------------->

	<!----------------------- just-relax --------------------------------->
	<div class="special-bg-blok">
		<div class="background-icon"></div>
		<div class="background-foto">
			<div class="special-bg-blok-content">
				<div class="heading-2 white">Just relax <em>&</em> let us do the heavy lifting</div>
				<p>Proin iaculis purus consequat sem cure  digni ssim. Donec porttitora entum suscipit  aenean rhoncus posuere odio in tincidunt.</p>
				<ul class="list-icons-check">
					<li>Initial Setup & Customizations</li>
					<li>Regular Updates</li>
					<li>Round The Clock Support</li>
				</ul>
				<a class="button" href="#">Learn more</a>
			</div>
		</div>
	</div>
        <!---------------------- // just-relax ---------------------------------->
<script src="js/jquery-3.3.1.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>
