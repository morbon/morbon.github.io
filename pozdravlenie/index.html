<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>С новым 2019 годом!</title>
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<head>
	<title></title>
</head>
<body>
	<div class="container">
		<p class="fig">
		  <img class="center" src="img/1546009306-2525.jpg">
		</p>
		<p class="text">
		 <strong>Желаю в этот Новый год<br>
			Поменьше грусти и забот,<br>
			Побольше счастья и добра,<br>
			Улыбок, нежности, тепла!</strong></p>


			<p class="text"><strong>Чтоб были верными друзья<br>
			И очень дружною семья,<br>
			Чтоб каждый день удачным был,<br>
			И чтоб на всё хватало сил!</strong></p>


		<p class="text"><strong>Ну а ещё пусть Новый год<br>
			Побольше денег принесёт,<br>
			Здоровья, мира и любви,<br>
			Чтоб в сердце не было зимы!</strong><br>
		</p>
	</div>
</body>
</html>
