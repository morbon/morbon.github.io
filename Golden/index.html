<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="utf-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
	<link rel="shortcut icon" href="img/favicon.png" type="image/png">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img class="logo" src="img/logo.png" alt="Golden">
				<nav>
					<ul class="menu">
						<li><a href="#">Home</a></li>
						<li><a href="#">Services</a></li>
						<li><a href="#">Portfolio</a></li>
						<li><a href="#">About</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles_first">Welcome To Our Studio!</div>
				<h1>It’s nice to Meet you</h1>
			</div>
			<a class="button" href="#">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="services clearfix">
				<div class="services__item">
					<img src="img/icon1.png" alt="Услуга">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services__item">
					<img src="img/icon2.png" alt="Услуга">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services__item">
					<img src="img/icon3.png" alt="Услуга">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt="Работа">
				<img src="img/img2.jpg" alt="Работа">
				<img src="img/img3.jpg" alt="Работа">
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>About Us</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="month clearfix">
				<div class="month_left-blok">
					<div class="title_mouth">
						<span>July 2010</span><br>
						<b>Our Humble Beginnings</b>
					</div>
					<p>
						Proin iaculis purus consequat sem cure
						digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt. Proin
						iaculis purus consequat sem cure digni
						ssim. Donec porttitora entum suscipit.
					</p>
				</div>
				<div class="month_left-blok">
					<img src="img/about1.jpg" alt="История">
				</div>
			</div>
			<div class="month clearfix">
				<div class="month_right-blok">
					<div class="title_mouth">
						<span>January 2011</span><br>
						<b>Facing Startup Battles</b>
					</div>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit aenean 
						rhoncus posuere odio in tincidunt.
					</p>
				</div>
				<div class="month_right-blok">
					<img src="img/about2.jpg" alt="История">
				</div>
			</div>
			<div class="month clearfix">
				<div class="month_left-blok">
					<div class="title_mouth">
						<span>December 2012</span><br>
						<b>Enter The Dark Days</b>
					</div>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni.
					</p>
				</div>
				<div class="month_left-blok">
					<img src="img/about3.jpg" alt="История">
				</div>
			</div>
			<div class="month clearfix">
				<div class="month_right-blok">
					<div class="title_mouth">
						<span>february 2014</span><br>
						<b>Our Triumph</b>
					</div>
					<p>
						Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit aenean.
					</p>
				</div>
				<div class="month_right-blok">
					<img src="img/about4.jpg" alt="История">
				</div>
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<div class="signature">
				<p><span>&copy</span> Copyright 2014 FreebiesXpress.com (layout Beatris)</p>
			</div>
		</div>
	</footer>
</body>
</html>